packages/cursorless-engine/src/processTargets/modifiers/scopeHandlers/CollectionItemScopeHandler/createTargetScope.ts

```diff
@@ -0,0 +1,54 @@
+import { type TextEditor, Range } from "@cursorless/common";
+import { getRangeLength } from "../../../../util/rangeUtils";
+import { ScopeTypeTarget } from "../../../targets";
+import type { TargetScope } from "../scope.types";
+
+export function createTargetScope(
+  isEveryScope: boolean,
+  editor: TextEditor,
+  iterationRange: Range,
+  contentRange: Range,
+  previousRange?: Range,
+  nextRange?: Range,
+): TargetScope {
+  const leadingDelimiterRange =
+    previousRange != null
+      ? new Range(previousRange.end, contentRange.start)
+      : undefined;
+  const trailingDelimiterRange =
+    nextRange != null
+      ? new Range(contentRange.end, nextRange.start)
+      : undefined;
+
+  // We have both leading and trailing delimiter ranges
+  // The leading one is longer/more specific so prefer to use that for removal.
```

Comment:
```suggestion
  // If the leading one is longer/more specific, prefer to use that for removal;
  // otherwise use undefined to fallback to the default behavior (often trailing)
```